kubermatic-addons/addon-manifests/kubelb.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kubermatic.k8s.io/v1
+kind: AddonConfig
+metadata:
+  name: kubelb
+spec:
+  description: Kubelb-Agent dynamically creates load balancers and external ingress controller implementation.
+  formSpec:
+    - displayName: Base64 encoded Kubeconf to access the kubelb cluster
```

Comment:
Better expect plaintext and do the b64 encoding yourself in the template. KKP seems to support sprig for that: http://masterminds.github.io/sprig/